Description: Remove the _template directory from the sphinx configuration.
 It was an old default that wasn't up-to-date. Since there was no
 customization, the best bet is to entirely remove it to avoid having the issue
 next time Sphinx changes something. This patch is NOT the one upstreamed but
 is only a workaround until the next release.
Author: Simon Chopin <chopin.simon@gmail.com>
Bug: http://github.com/pazz/alot/issues/503
--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -57,7 +57,7 @@
 
 
 # Add any paths that contain templates here, relative to this directory.
-templates_path = ['_templates']
+templates_path = []
 
 # The suffix of source filenames.
 source_suffix = '.rst'
